Formula/clang-format.rb

```diff
@@ -4,10 +4,10 @@ class ClangFormat < Formula
   version "2016-12-09"
 
   stable do
-    url "http://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
+    url "https://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
```

Comment:
Ah. I can't quite figure that one out, given that secure SF SVN connections & various others still remain busted using the system, but yeah, I guess it makes sense if anything would work the newest macOS versions would.